platform/android/README.md

```diff
@@ -8,11 +8,9 @@ A library based on [Mapbox GL Native](../../README.md) for embedding interactive
 
 Alright. So, actually, you may be in the wrong place. From here on in, this README is going to be for people who are interested in working on and improving on Mapbox GL Native for Android.
 
-**To view our current API documentation, see our [JavaDoc](https://www.mapbox.com/android-sdk/api).**
+**To view our current API documentation, see our [JavaDoc](https://www.mapbox.com/android-docs/api/map-sdk/5.0.2/index.html).**
 
```

Comment:
I'll look into adding a more generic URL, opened a ticket for tracking: https://github.com/mapbox/android-docs/issues/123